subsys/usb/device/class/audio/Kconfig

```diff
@@ -19,4 +19,25 @@ module = USB_AUDIO
 module-str = USB Audio
 source "subsys/logging/Kconfig.template.log_config"
 
+config USB_DEVICE_AUDIO_MIN_VOLUME
+	hex "USB Audio Device Min Volume"
+	default 0
+	range 0 32767
+	help
+	  attributes can range from 32767 down to 0
```

Comment:
It is somewhat inconsistent that the parameter is of hex type and descriptor uses decimal.

Reply:
> It is somewhat inconsistent that the parameter is of hex type and descriptor uses decimal.

Thanks, I changed to hex type.